viorng/viorng/viorng.inf

```diff
@@ -90,21 +89,16 @@ HKR,Parameters,DmaRemappingCompatible,0x00010001,1
 [VirtRng_Provider_CopyFiles]
 viorngum.dll
 
-; -------------------------------------
-; WDF & Device CoInstaller Installation
-; -------------------------------------
-
-[VirtRng_Device.NT.CoInstallers]
-AddReg=VirtRng_Wdf_CoInstaller_AddReg
-CopyFiles=VirtRng_Wdf_CoInstaller_CopyFiles
+[VirtRng_Provider_AddReg]
+HKLM,SYSTEM\CurrentControlSet\Control\Cryptography\Providers\%PROVIDER_NAME%\UM,Image,,viorngum.dll
+HKLM,SYSTEM\CurrentControlSet\Control\Cryptography\Providers\%PROVIDER_NAME%\UM\00000006,Flags,%REG_DWORD%,0x00000001
+HKLM,SYSTEM\CurrentControlSet\Control\Cryptography\Providers\%PROVIDER_NAME%\UM\00000006,Functions,%REG_MULTI_SZ%,RNG
+HKLM,SYSTEM\CurrentControlSet\Control\Cryptography\Configuration\Local\Default\00000006\RNG,Providers,%REG_MULTI_SZ_APPEND%,%PROVIDER_NAME%
 
-[VirtRng_Wdf_CoInstaller_AddReg]
-HKR,,CoInstallers32,0x00010000,"viorngci.dll,VirtRngCoInstaller", \
-    "WdfCoInstaller$KMDFCOINSTALLERVERSION$.dll,WdfCoInstaller"
 
-[VirtRng_Wdf_CoInstaller_CopyFiles]
-WdfCoInstaller$KMDFCOINSTALLERVERSION$.dll
-viorngci.dll
+; -------------------------------------
+; WDF
```

Comment:
Why did you remove the WDF coinstaller `WdfCoInstaller$KMDFCOINSTALLERVERSION$.dll` ?

Reply:
Since all work is done in the `AddReg` directive and there is no coinstaller, I saw no reason to include this in the INF file.

Reply:
Ah, I see, this is for the correct WDF version to install. I will bring it back.